scripts/sparse-and-shallow-git-fetch.sh

```diff
@@ -10,7 +10,8 @@ if [ "$#" -ne 3 ]; then
   exit 1
 fi
 
-set -ev
+# Exit immediately if command exits with non-zero status.
+set -e
```

Comment:
It can be good for debugging argument passing. I don't know if that was the intent here or not.